_faq/high_involvement_status.md

```diff
@@ -0,0 +1,11 @@
+---
+question: "What is the meaning of High Involvement status of any project?"
+---
+
+It simply means that the project already has more than one proposal and if you want to submit the proposal <br>
```

Comment:
break lines at 80cpl

do not use `<br>` in markdown

Reply:
May I use `\` to break the line?